icu4c/source/common/unicode/ucurr.h

```diff
@@ -113,7 +113,26 @@ typedef enum UCurrNameStyle {
      *
      * @stable ICU 61
      */
-    UCURR_NARROW_SYMBOL_NAME
+    UCURR_NARROW_SYMBOL_NAME,
+
+    /**
```

Comment:
In public header files, you need to wrap `@draft` APIs with `U_HIDE_DRAFT_API`.  Search for examples.